tests/RangeTest.java

```diff
@@ -0,0 +1,157 @@
+import org.junit.*;
+import util.*;
+
+public class RangeTest {
+
+    @Test
+    public void Contains() throws Exception {
+        Range range1;
+        //One Dimension
+        range1 = Range.OneDimensionalRange(1,4);
+        //Contains
+        Assert.assertTrue(range1.contains(1));
+        Assert.assertTrue(range1.contains(3));
+        //Not Contains
+        Assert.assertFalse(range1.contains(0));
+        Assert.assertFalse(range1.contains(5));
+        //Two Dimension
+        range1 = Range.TwoDimensionalRange(0, 20, 0, 20);
+        //Contains
+        Assert.assertTrue(range1.contains(0, 0));
+        Assert.assertTrue(range1.contains(20, 20));
+        Assert.assertTrue(range1.contains(10, 10));
+        //Not Contains
+        Assert.assertFalse(range1.contains(0, 21));
+        Assert.assertFalse(range1.contains(21, 20));
+    }
+
+    @Test
+    public void OneDimensionEquals() throws Exception {
+        Range range1 = Range.OneDimensionalRange(0, 20);
+        Range range2;
+        //Equals
+        range2 = Range.OneDimensionalRange(0, 20);
+        Assert.assertTrue(range1.equals(range2));
```

Comment:
You could use Assert.assertEquals here. It'll call your overrided equals method.